### PR TITLE
Basic email system: Big overhaul of grammar and formatting (Revision 2)

### PR DESCRIPTION
**Pull request by neelchauhan**
It seemed my previous PR #1854 got prematurely merged and subsequently reverted in #1863, so here is a revised version.

#### Author checklist (Completed by original Author)
- [X] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [X] If applicable, steps and instructions have been tested to work
- [X] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

**Comment by sspencerwire** [comment]
@jimcat8 Please review these changes. @wsoyinka I'm not going to approve or merge this PR until I know that it is in its final state. @neelchauhan Thank you for your work!

File: docs/guides/email/02-basic-email-system.md [hunk 1]
```diff
@@ -57,10 +63,11 @@ zone "rockylinux.me" IN {
 
 !!! question 
 
-    **What is DNS zone?** A DNS zone is the specific portion of a DNS namespace that's hosted on a DNS server. A DNS zone contains resource records, and the DNS server responds to queries for records in that namespace. A DNS server can have multiple DNS zones. Simply put, a DNS zone is the equivalent of a book catalog.
+    **What is a DNS zone?** A DNS zone is aportion of the DNS namespace that's hosted on a DNS server. A DNS zone contains resource records, and a DNS server responds to queries for records in that namespace. A DNS server can have multiple DNS zones. Simply put, a DNS zone is analogous to a book catalog.
+
+First, initalize BIND:
```
Comment (SergeCroise) [inline docs/guides/email/02-basic-email-system.md:68]:
```suggestion
First, initialize BIND:
```

File: docs/guides/email/02-basic-email-system.md [hunk 2]
```diff
@@ -210,32 +232,30 @@ Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(1,1,
 Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(2,1,'all@mail.rockylinux.me','leeo@mail.rockylinux.me');
 ```
 
-Here I have not inserted the ciphertext password for the relevant email users, which requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
+The password entries for the relevant email users are not shown here, as it requires the use of `doveadm pw -s SHA512-crypt -p twotestandtwo` command. See [here](#ap1)
```
Comment (SergeCroise) [inline docs/guides/email/02-basic-email-system.md:235]: As a translator, I would prefer to put the command in a separate line, e.g., something similar to:

The password entries for the relevant email users are not shown here, as it requires the use of the following command:
```bash
doveadm pw -s SHA512-crypt -p twotestandtwo
```
 See [here](#ap1).

What is the meaning of the other translators?

File: docs/guides/email/02-basic-email-system.md [hunk 3]
```diff
@@ -248,22 +268,22 @@ Use the $ sign to separate the output text information.
 ### Install and configure `postfix`
 
 ```bash
-Shell(192.168.100.6) > dnf -y install postfix postfix-mysql
+dnf -y install postfix postfix-mysql
 ```
 
 After installing Postfix, the following files need to be know:
```
Comment (SergeCroise) [inline docs/guides/email/02-basic-email-system.md:274]:
```suggestion
After installing Postfix, the following files need to be known:
```

File: docs/guides/email/02-basic-email-system.md [hunk 4]
```diff
@@ -621,35 +670,55 @@ userdb {
 
 !!! warning
```
Comment (SergeCroise) [inline docs/guides/email/02-basic-email-system.md:671]:
```suggestion
!!! warning "Warning"
```

File: docs/guides/email/02-basic-email-system.md [hunk 5]
```diff
@@ -678,16 +747,20 @@ service auth-worker {
...
 ```
 
-OK, use the command to start your service-- `systemctl start dovecot.service`
+Next enable Dovecot:
+
+```bash
+Shell(192.168.100.7) > systemctl enable --now dovecot
+```
 
 !!! info
```
Comment (SergeCroise) [inline docs/guides/email/02-basic-email-system.md:756]:
```suggestion
!!! info "Info"
```

**Review by SergeCroise — COMMENTED** [review]
(feel free to modify or revert my changes)

File: docs/guides/email/02-basic-email-system.md (hunk 5 above)
Comment (sspencerwire) [reply docs/guides/email/02-basic-email-system.md:756]: We have discussed this before @SergeCroise, but adding the "Info" and "Warning" to custom titles isn't required. In fact, I think we discussed that translators could decide if they wanted to add the translated title to the admonition themselves. If I recall, that was the consensus decision, wasn't it?

**Comment by sspencerwire** [comment]
@jimcat8: What is it that you don't like about the fenced code-block changes?  The title of the fenced code block is a required markdown element (```bash) so I'm guessing that this is not what you are arguing about?  Please clarify for @neelchauhan what you don't want.

File: docs/guides/email/02-basic-email-system.md (hunk 1 above)
Comment (sspencerwire) [reply docs/guides/email/02-basic-email-system.md:68]: @SergeCroise Above, you have "A DNS zone is aportion of..." If you are suggesting a change, it should be "A DNS zone is a portion of...", two words.

File: docs/guides/email/02-basic-email-system.md [hunk 6]
```diff
@@ -619,37 +668,57 @@ userdb {
...
-!!! warning
+!!! warning "Warning"
+
+    Don't write the above grammar on one line, such as this: `userdb {driver = sql args = uid=vmail gid=vmail home=/var/mail/vhosts/%d/%n}`. Otherwise, it won't work.
+
+Edit the `/etc/dovecot/dovecot-sql.conf.ext` file:
 
-    Don't write the above grammar on one line, such as this--"userdb {driver = sql args = uid=vmail gid=vmail home=/var/mail/vhosts/%d/%n}". Otherwise, it won't work.
+```bash
+Shell(192.168.100.7) > vim /etc/dovecot/dovecot-sql.conf.ext
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:678]: @jimcat8  and @neelchauhan  

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 7]
```diff
@@ -545,64 +572,86 @@ Shell(192.168.100.6) > tree /etc/dovecot/
...
-...
 protocols = imap pop3 lmtp
 listen = 192.168.100.6
-...
 ```
 
+Next, edit the mail storage configuration:
+
+```bash
+Shell(192.168.100.7) > vim /etc/dovecot/conf.d/10-mail.conf
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:621]: @jimcat8  and @neelchauhan  

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 8]
```diff
@@ -545,64 +572,86 @@ Shell(192.168.100.6) > tree /etc/dovecot/
...
+* `login_trusted_networks= `: Which IP networks are allowed to use Dovecot. It can be a single IP address, a network segment, or both. As an example: `login_trusted_networks = 10.1.1.0/24 192.168.100.2`
+* `mail_location = `: For an empty value, Dovecot attempts to find the mailboxes automatically (looking at `~/Maildir`, `/var/mail/username`, `~/mail`, and `~/Mail`, in that order). However, auto-detection commonly fails for users whose mail directory hasn’t yet been created, so you should explicitly state the full location here, if possible.
+* `mail_privileged_group = `: This group is enabled temporarily for privileged operations. Currently, this is used only with the INBOX when either its initial creation or dotlocking fails. Typically, this is set to `mail` to access `/var/mail`.
 
 #### Modify multiple files
 
+First, edit the Dovecot configuration:
+
+```bash
+Shell(192.168.100.7) > vim /etc/dovecot/dovecot.conf
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:608]: @jimcat8  and @neelchauhan  . Especially @jimcat8 

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 9]
```diff
@@ -500,21 +527,21 @@ smtps     inet  n       -       n       -       -       smtpd
...
+dnf config-manager --enable devel
+dnf -y install dovecot dovecot-devel dovecot-mysql
 ```
 
-Without changing any files, the original directory structure is as follows:
+Without changing any files, the default Dovecot directory structure is as follows:
 
 ```bash
-Shell(192.168.100.6) > tree /etc/dovecot/
+Shell(192.168.100.7) > tree /etc/dovecot/
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:544]: @jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 10]
```diff
@@ -448,34 +479,30 @@ query = SELECT email FROM virtual_users WHERE email='%s'
...
-    If you encounter this kind of error after running `systemctl start postfix.service` -- "fatal: open lock file /var/lib/postfix/master.lock: unable to set exclusive lock: Resource temporarily unavailable." Please delete the existing **/var/lib/postfix/master.lock** file
+    If you encounter this kind of error after running `systemctl start postfix.service`: `fatal: open lock file /var/lib/postfix/master.lock: unable to set exclusive lock: Resource temporarily unavailable.` Please delete the existing `/var/lib/postfix/master.lock` file
 
 Testing Postfix configure:
 
 ```bash
-Shell(192.168.100.6) > systemctl start postfix.service
-Shell(192.168.100.6) > postfix check
-Shell(192.168.100.6) > postfix status
+Shell(192.168.100.7) > systemctl start postfix.service
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:487]: @jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 11]
```diff
@@ -210,32 +232,30 @@ Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(1,1,
...
-Shell(192.168.100.5) > grep -i method /etc/login.defs
+Shell(192.168.100.7) > grep -i method /etc/login.defs
 ENCRYPT_METHOD SHA512
 ```
 
 We can see its structure in the /etc/shadow file:
 
 ```bash
-Shell(192.168.100.5) > grep -i root /etc/shadow | cut -f 2 -d ":"
+Shell(192.168.100.7) > grep -i root /etc/shadow | cut -f 2 -d ":"
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:258]: @jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.5 (original) OR 192.168.100.7 (new) ?

File: docs/guides/email/02-basic-email-system.md [hunk 12]
```diff
@@ -210,32 +232,30 @@ Mysql > insert into virtual_aliases(id,domain_id,source,destination) values(1,1,
...
 
-In the SHA-2 encryption standard, the number in the algorithm refers to the digest length in bits.
+In the SHA-2 hashing standard, the number in the algorithm refers to the digest length in bits.
 
-It is well known that in RockyLinux 8 and other RHEL 8 variants, the algorithm used to encrypt user passwords is SHA-512.
+It is well known that in Rocky Linux 8 and other RHEL 8 variants, the algorithm used to encrypt user passwords is SHA-512.
 
 ```bash
-Shell(192.168.100.5) > grep -i method /etc/login.defs
+Shell(192.168.100.7) > grep -i method /etc/login.defs
```
Comment (wsoyinka) [inline docs/guides/email/02-basic-email-system.md:251]: @jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.5 (original) OR 192.168.100.7 (new) ?

**Comment by jimcat8** [comment]
My idea is not to adjust the basic style or format, but only to change the wording of the text, because the content of this document (02-Basic-Email-Syst.md) is relatively complete.
Just like this “PR” (# 1865)

**Review by wsoyinka — COMMENTED** [review]
@jimcat8  :

-  Please review and confirm that specific IP address changes at the prompt are correct throughout the document(s). 

192.168.100.5 Vs  192.168.100.6 Vs.  192.168.100.7

File: docs/guides/email/02-basic-email-system.md (hunk 6 above)
Comment (jimcat8) [reply docs/guides/email/02-basic-email-system.md:678]: > @jimcat8 and @neelchauhan
> 
> * what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

The beginning of the document already indicates that this is an email system composed of three machines.

File: docs/guides/email/02-basic-email-system.md (hunk 12 above)
Comment (neelchauhan) [reply docs/guides/email/02-basic-email-system.md:251]: 192.168.100.5

File: docs/guides/email/02-basic-email-system.md (hunk 11 above)
Comment (neelchauhan) [reply docs/guides/email/02-basic-email-system.md:258]: 192.168.100.5

File: docs/guides/email/02-basic-email-system.md (hunk 9 above)
Comment (neelchauhan) [reply docs/guides/email/02-basic-email-system.md:544]: 192.168.100.6

File: docs/guides/email/02-basic-email-system.md (hunk 8 above)
Comment (neelchauhan) [reply docs/guides/email/02-basic-email-system.md:608]: 192.168.100.6

File: docs/guides/email/02-basic-email-system.md (hunk 7 above)
Comment (neelchauhan) [reply docs/guides/email/02-basic-email-system.md:621]: 192.168.100.6

**Comment by sspencerwire** [comment]
@neelchauhan @jimcat8 Because of the errors made in the IP addresses of the prompts, and because this PR now conflicts with a grammar PR pushed and merged yesterday (which did not deal with the technical aspects of this document), I'm going to have to elect to close this PR yet again. @neelchauhan If you see any wording changes that still need to be dealt with, without messing with the technical aspects of the document, I'm more than willing to merge those. Thanks to all.

**Pull request closed without merging**